src/main/java/uk/gov/hmcts/reform/unspec/validation/RequestExtensionValidator.java

```diff
@@ -0,0 +1,63 @@
+package uk.gov.hmcts.reform.unspec.validation;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+import java.util.List;
+
+import static java.time.LocalDate.now;
+import static java.util.Collections.emptyList;
+import static uk.gov.hmcts.reform.unspec.handler.RequestExtensionCallbackHandler.PROPOSED_DEADLINE;
+import static uk.gov.hmcts.reform.unspec.handler.RequestExtensionCallbackHandler.RESPONSE_DEADLINE;
+
+@Service
+@RequiredArgsConstructor
+public class RequestExtensionValidator {
+
+    public static final LocalTime END_OF_COURT_TIME = LocalTime.of(16, 0);
```

Comment:
is end of court time correct? end of working day?

Reply:
yes that's correct.

Reply:
end of court time is the business term?